packages/tenant-process/src/routers/TenantRouter.ts

```diff
@@ -178,7 +180,8 @@ const tenantsRouter = (
               .status(404)
               .json(
                 makeApiProblem(
-                  tenantNotFound(`${origin}/${code}`),
+                  // TODO create a dedicated error. origin/code is not a valid TenantId
```

Comment:
Same question here, scope of this PR or tracked on Jira?

Reply:
Not tracked! Maybe @taglioni-r or @Carminepo2 could investigate.

Reply:
I created a dedicated function `tenantFromExternalIdNotFound` to create this kind of error